cluster-scope/base/user.openshift.io/groups/testgroup/group.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: user.openshift.io/v1
```

Comment:
this entire `cluster-scope` folder I'm guessing was not supposed to be included in this pr

Reply:
you are correct, removing it.